sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/models/HttpRequestContext.java

```diff
@@ -384,4 +386,41 @@ public String getParameterName() {
             return parameterName;
         }
     }
+
+    /**
+     * Represents a query parameter.
+     */
+    public static class QueryParameter {
+        private final String value;
+        private final boolean isMultiple;
```

Comment:
We should also consider including the delimiter here and default to `,` if it is `null`.